packages/api-queries/src/site-scan.ts

```diff
@@ -9,6 +11,12 @@ import {
 import { mutationOptions, queryOptions } from '@tanstack/react-query';
 import { queryClient } from './query-client';
 
+export const bulkFixThreatsStatusQuery = ( siteId: number, threatIds: number[] ) =>
```

Comment:
Same here, could we name it `fixThreatsStatusQuery`. We will also use it to validate the progress status of an individual threat.

Reply:
Updated it.